src/components/extensions/Scopes.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { SCOPES } from "../../constant";
+
+const HeadingLink = styled.a`
```

Comment:
I'm wondering if there is any way we can pull these styles from the original css file, but I don't see any obvious ways to this. We'd need to remember about this piece whenever we are changing styles (hopefully not that often)